app/models/screening.rb

```diff
@@ -3,7 +3,8 @@ class Screening < ApplicationRecord
   belongs_to :hall
   has_many :reservations
   validates :start_time, :end_time, presence: true
-  validates_comparison_of :end_time, greater_than: :start_time, greater_than: Time.now
+  validates_comparison_of :end_time, greater_than: :start_time
+  validates :start_time, comparison: {greater_than: Time.now.in_time_zone('Poland')}
```

Comment:
In the project with a lot of validatons you can be lost searching for a specific one.
What do you think about keeping all validations per attribute. Same with `end_time`

```suggestion
  validates :start_time, comparison: {greater_than: Time.now.in_time_zone('Poland')}, presence: true
```

Reply:
Also here you have two same types of validations (comparison) and two different syntax for them. It is not a mistake, but it is better to KISS. It will be easier for human eye to parse.